packages/kit/test/apps/dev-only/package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "test-basics",
+	"name": "fack-off",
```

Comment:
lol
```suggestion
	"name": "dev-only",
```

Reply:
LOL, I feel like I need to explain this: The name was the same as the `basics` test app so I can't run `pnpm -F test-basics test`. I've had to change it across three PRs now (I fixed it in another PR). Thanks for this catch <picture data-single-emoji=":lol:" title=":lol:"><img class="emoji" width="20" height="auto" src="https://emoji.slack-edge.com/T0CAQ00TU/lol/5986e156eca2154f.png" alt=":lol:" align="absmiddle"></picture>